<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1069</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_18">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QFrame" name="Menu">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>910</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>12</y>
           <width>177</width>
           <height>961</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>171</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>181</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;DASHBOARD&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#eeeeec;&quot;&gt;Packet Count: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="no_packets">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#eeeeec;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="minimumSize">
                 <size>
                  <width>171</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="text">
                 <string>Settings</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="minimumSize">
                 <size>
                  <width>171</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="text">
                 <string>ADD</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="minimumSize">
                 <size>
                  <width>171</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="text">
                 <string>CONFIG FILE</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>254</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>254</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#000000;&quot;&gt;Command List&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="commandslist">
                <property name="minimumSize">
                 <size>
                  <width>256</width>
                  <height>192</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>265</width>
                  <height>193</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="request">
                <property name="minimumSize">
                 <size>
                  <width>256</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>265</width>
                  <height>25</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Post</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Get</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="minimumSize">
                   <size>
                    <width>254</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>263</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Send</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>256</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>265</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Response&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="Response">
                <property name="minimumSize">
                 <size>
                  <width>256</width>
                  <height>217</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>265</width>
                  <height>256</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="minimumSize">
             <size>
              <width>3</width>
              <height>529</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QFrame" name="horizontalFrame">
              <property name="minimumSize">
               <size>
                <width>751</width>
                <height>550</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout"/>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>58</width>
                    <height>36</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>65</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Min Time</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="doubleSpinBox">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="minimumSize">
                       <size>
                        <width>42</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>Y-Axis</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDoubleSpinBox" name="yaxisspinner">
                      <property name="minimumSize">
                       <size>
                        <width>50</width>
                        <height>30</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="minimumSize">
                     <size>
                      <width>42</width>
                      <height>32</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>X-Axis</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="xaxisspinner">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>50</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QComboBox" name="apbox"/>
              </item>
              <item>
               <widget class="QComboBox" name="uebox"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QCheckBox" name="checkBox_2">
                  <property name="text">
                   <string>Autoscale</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox">
                  <property name="text">
                   <string>PointLabels</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="graphleft">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&lt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="graphright">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="minimumSize">
           <size>
            <width>1030</width>
            <height>3</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QFrame" name="horizontalFrame">
                <property name="minimumSize">
                 <size>
                  <width>401</width>
                  <height>220</height>
                 </size>
                </property>
                <layout class="QHBoxLayout" name="cpuframe"/>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>57</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>CPU % - </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QProgressBar" name="cpubar">
                  <property name="minimumSize">
                   <size>
                    <width>81</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>81</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="value">
                   <number>24</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="minimumSize">
                   <size>
                    <width>249</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="minimumSize">
               <size>
                <width>3</width>
                <height>270</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="verticalFrame">
              <property name="minimumSize">
               <size>
                <width>461</width>
                <height>250</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>250</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="qview"/>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="statusbar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="Line" name="line_4">
              <property name="minimumSize">
               <size>
                <width>1028</width>
                <height>3</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1282</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionInfo"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
